source/blog/_posts/2016-07-14-open-hackathon-review.md

```diff
@@ -0,0 +1,70 @@
+You can find most of the code that was written during the hackathon on [GitHub](https://github.com/ShopwareHackathon).
+
+I will not describe every project in detail here.
+
+### Shopware Plugin for PhpStorm
+A team lead by [Daniel Espendiller](https://twitter.com/BigHaehnchen) worked on new features for the [Shopware Plugin for PhpStorm](https://plugins.jetbrains.com/plugin/7410?pr=).
+This resulted in some cool features like initial support for the new plugins system and a project installer to download and unpack shopware directly from within PhpStorm.
+
+![](/blog/img/open-hackathon-2016/phpstorm-plugin.jpg)
+
+### Code name m2e (Meal to employee)
+Since a few weeks we have employee catering here at the Shopware HQ. But up to now the ordering process involves big spreadsheet in confluence (yikes!).
+To slimline the whole ordering process a RESTfuel API based on an existing Symfony application was created that is operated with a nice Angular based frontend.
+In the future this application will be also used to let attendees of our academy choose their meals.
+
+![](/blog/img/open-hackathon-2016/m2e_erd.jpg)
+
+### Unsplash
+Another team integrated [unplash](https://unsplash.com/) into the shopware media manger. Unsplash provides Do-whatever-you-want high-resolution photos via an API. This API was integrated into the media manager so you can find photos for emotion worlds etc. from unsplash.
+
+![](/blog/img/open-hackathon-2016/unsplash.png)
+<div style="clear:both"></div>
+
+### Meleven Image Cloud Integration
+<img src="/blog/img/open-hackathon-2016/meleven-logo.png" class="is-float-left" style="width:175px" />
+A few people from our partner shopmacher worked on an integration to the [meleven](http://www.meleven.de/) image cloud. This goes further that our current CDN support as all thumbnails are generated on-the-fly by meleven.
+
+<div style="clear:both"></div>
+### Plugin System Developer Experience Improvements
+Improve the Developer Experience of the [new plugin system](developers-guide/plugin-system/) was another project that will have direct impact for shopware plugin developers. This involved thing like automatic registration of JS/LESS files or a more convenient way to register customer controllers.
+
+### Shopware goes composer (part 2)
+<img src="/blog/img/open-hackathon-2016/logo-composer-transparent4.png" class="is-float-left" style="width:175px" />
+
+During the last (internal) hackathon I worked on a project called [Shopware goes Composer](blog/2016/02/11/projects-of-the-first-internal-hackathon-in-2016/#shopware-goes-composer). Goal of this project is to deploy shopware via git as a composer dependency as well as install plugins via composer. During this hackathon the team did great progress to reach this goal. There are still a lot of patches that need to be merged in the main shopware branch but "soon" it should be possible to deploy shopware from git.
```

Comment:
During the last (internal) hackathon, I worked on a project called [Shopware goes Composer](blog/2016/02/11/projects-of-the-first-internal-hackathon-in-2016/#shopware-goes-composer), the goal of which was to deploy Shopware via git as a composer dependency as well as install plugins via composer. During this hackathon, the team made great progress to reach this goal. There are still a lot of patches that need to be merged in the main shopware branch but "soon" it should be possible to deploy Shopware from git.